functions/package-lock.json

```diff
@@ -5307,9 +5307,9 @@
       }
```

Comment:
As there are no changes in the `package.json` now, it would be good to revert the changes in this file as well.

Reply:
Thanks! Just pushed a commit that should take care of that